packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -20,19 +23,19 @@ import { v4 as uuid } from "uuid";
  */
 export class InsecureTinyliciousUrlResolver implements IUrlResolver {
```

Comment:
why not just add a construcutor and a param to make the whole address configurable? you can default to localhost and that port